website/docs/r/api_management.html.markdown

```diff
@@ -192,6 +192,8 @@ In addition to all arguments above, the following attributes are exported:
 
 * `portal_url` - The URL for the Publisher Portal associated with this API Management service.
 
+* `public_ip_adresses` - The Public IP adresses of the API Management Service.
```

Comment:
```suggestion
* `public_ip_adresses` - The Public IP addresses of the API Management Service.
```

Reply:
Corrected the mistake in the latest commit